js/ui/html_editor/modules/tableResizing.js

```diff
@@ -0,0 +1,491 @@
+import $ from '../../../core/renderer';
+import eventsEngine from '../../../events/core/events_engine';
+import { isDefined } from '../../../core/utils/type';
+import { addNamespace } from '../../../events/utils/index';
+import _windowResizeCallbacks from '../../../core/utils/resize_callbacks';
+import { move } from '../../../animation/translator';
+import { getBoundingRect } from '../../../core/utils/position';
+import BaseModule from './base';
+import Draggable from '../../draggable';
+import { each } from '../../../core/utils/iterator';
+import { getWindow, hasWindow } from '../../../core/utils/window';
+import { extend } from '../../../core/utils/extend';
+
+const DX_COLUMN_RESIZE_FRAME_CLASS = 'dx-table-resize-frame';
+const DX_COLUMN_RESIZER_CLASS = 'dx-htmleditor-column-resizer';
+const DX_ROW_RESIZER_CLASS = 'dx-htmleditor-row-resizer';
+const DEFAULT_MIN_COLUMN_WIDTH = 40;
+
+const TIMEOUT = 100;
+
+const DRAGGABLE_ELEMENT_OFFSET = 2;
+
+const MODULE_NAMESPACE = 'dxHtmlTableResizingModule';
+
+const POINTERDOWN_EVENT = addNamespace('dxpointerdown', MODULE_NAMESPACE);
+const SCROLL_EVENT = addNamespace('scroll', MODULE_NAMESPACE);
+
+export default class TableResizingModule extends BaseModule {
+    constructor(quill, options) {
+        super(quill, options);
+        this.enabled = !!options.enabled;
+        this._tableResizeFrames = [];
+        this._minColumnWidth = options.minColumnWidth ?? DEFAULT_MIN_COLUMN_WIDTH;
+        this._minRowHeight = options.minRowHeight ?? DEFAULT_MIN_COLUMN_WIDTH / 2;
+        this._quillContainer = this.editorInstance._getQuillContainer();
+        this._quillInstance = quill;
+
+        if(this.enabled) {
+            this._attachResizerTimeout = setTimeout(() => {
```

Comment:
Can we use [addContentInitializedCallback ](https://github.com/DevExpress/DevExtreme/blob/67f962b4a6b228f7fd25cd81ea38e69871082b6b/js/ui/html_editor/ui.html_editor.js#L525)instead of using timeout?